python/apps/taiga/src/taiga/workflows/api/__init__.py

```diff
@@ -79,19 +80,6 @@ async def get_workflow(
     return await workflows_services.get_workflow_detail(project_id=id, workflow_slug=workflow_slug)
 
 
-################################################
```

Comment:
this file had 2 parts:
- workflows (api + misc)
- workflow status (api + misc)

so the "misc" of workflows is better next to workflow than misc.

Reply:
Understood, I'll also doubted to group it or not. I'll divide it in two.